package.json

```diff
@@ -91,7 +91,7 @@
     "prismjs": "^1.30.0",
     "rollup": "^2.79.1",
     "sucrase": "^3.35.0",
-    "vite": "npm:rolldown-vite@^7.1.4",
+    "vite": "^7.1.7",
```

Comment:
`rolldown-vite` is has a good handful of known issues w/ `manualChunks`, which we make use of during prerendering. Given that the speed up is basically negligible anyhow, I think switching back is for the best.